### ISSUE TITLE
i18n: check original english texts carefully

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbase/vorta#136]
if we have to change original texts, all translators will have to update their translations, so better do this before it gets to the translators. agreeing on some terminology and checking for consistency everywhere is also useful (e.g. don't call it archive here and snapshot there if you mean the same thing).

for this:
- the english original texts should be exported (to a .ts file) and spell and grammar checked by a good native speaker. this includes stuff like upper / lowercase and punctuation.
- make a glossary for better consistency?
- there are some places using `doing foo..` and some `doing foo...` - use 3 dots consistently.

**Comment by ThomasWaldmann** [comment]
- fix "Password" to "Password:" (init new backup repo dialogue)

**Comment by m3nu** [comment]
That would be my mess.

**Comment by ThomasWaldmann** [comment]
The hint about "FUSE for macOS" is too specific / not helpful on other OS.

**Comment by ThomasWaldmann** [comment]
The longer text shown when adding a repo talks about using a thumb drive for full backups.
While it is only an example, I'ld like to change that (not that someone takes this as a recommendation or "the usual way").

I think we should advise using a 2.5" USB3 HDD for full backups:
- the usually quite big capacity allows for full backups without the risk of being too small
- it does not require a separate power supply (compared to 3.5" USB HDDs)
- it is not lost as easily as a thumbdrive
- it doesn't die as often as thumbdrives (from my experience, YMMV)

**Comment by m3nu** [comment]
> The hint about "FUSE for macOS" is too specific / not helpful on other OS.

It's [only shown](https://github.com/borgbase/vorta/blob/4f6137207b7580dda754f24c1243484a592e31f9/src/vorta/views/archive_tab.py#L39) for macOS. Nobody else will see it.

**Comment by m3nu** [comment]
> I think we should advise using a 2.5" USB3 HDD for full backups:

Makes sense. Change it to:

> Backup profiles allow for granular backups from different sources to different destinations. You could e.g. back up essential documents to a remote repository via Wifi, while doing a full backup onto a local **2.5" hard drive or thumb drive**.

**Comment by ThomasWaldmann** [comment]
BTW, these original string changes can be done in a separate PR, after #134 is merged. It already starts to diverge (see wording "snapshot" vs. "archive") with other PR.

**Comment by m3nu** [comment]
Ok. Stopping now. I also worry about conflicts with #134.

**Comment by ThomasWaldmann** [comment]
Another issue i noticed are the prune options (keep d/w/m/y):

there should be a little space between the text and the following control item, also between the control item and the following text.

not sure if this should be some space character in the original string or if the control item could have some sort of horizontal padding (if possible, guess that is the better solution than having leading/trailing space in strings).
